Gemfile

```
@@ -8,6 +8,7 @@ gemspec path: "examples/rust_reverse"
 gem "rake", "~> 13.0"
 gem "minitest", "5.15.0"
 gem "rake-compiler", "~> 1.2.5" # Small bug in 1.2.4 that breaks Ruby 2.5
+gem "rake-compiler-dock", "~> 1.9.1" # This should match the versions used in docker/Dockerfile.*
```

Nit, but shouldn't this be

```suggestion
gem "rake-compiler-dock", "1.9.1" # This should match the versions used in docker/Dockerfile.*
```

that way the version is absolutely pinned to the Dockerfile.

---

Yes, good point, thanks.